omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@nimid Im thinking in a way that `classes/charge.php` is a controller,

Now understand that why you want to have "get*()" inside `Charge.php` itself.

But then, now I don't see the benefit to have delegated class.
It just did move "logic & stuff" from main class to delegated class, but at the end, what you've concerned will happen at that delegated class anyway.